### ISSUE TITLE
[1.4-beta.9] EPERM issues on windows

### ISSUE THREAD

**Issue by sebakerckhof** [meteor/meteor#7391]
On Windows 10, I created a new app using `meteor create . --release 1.4-beta.9`
When I try to run it I always get:
```
C:\Users\seba\AppData\Local\.meteor\packages\meteor-tool\1.4.0-beta.9\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:165
      throw error;
      ^

Error: EPERM: operation not permitted, rename 'C:\Users\seba\Documents\GitHub\14test\.meteor\dev_bundle-uu8xy5647nhoajor' -> 'C:\Users\seba\Documents\GitHub\14test\.meteor\dev_bundle'
    at Error (native)
    at Object.fs.renameSync (fs.js:681:18)
    at exports.makeLink (C:\tools\cli\dev-bundle-links.js:16:6)
    at [object Object].ensureDevBundleLink (C:\tools\project-context.js:1370:5)
    at [object Object]._readFile (C:\tools\project-context.js:1329:10)
    at new exports.ReleaseFile (C:\tools\project-context.js:1279:8)
    at C:\tools\cli\main.js:817:22
```

Tried creating another app, same problem.

**Comment by mjmasn** [comment]
I think this is possibly already fixed in 1.3.4.4, so should make it into the 1.4-beta soon (See: https://github.com/meteor/meteor/issues/7374#issuecomment-231566566).

**Comment by benjamn** [comment]
Does `C:\Users\seba\Documents\GitHub\14test\.meteor\dev_bundle` already exist?

**Comment by sebakerckhof** [comment]
Well, I started from scratch, so nothing existed. 
But meteor has indeed already created a `dev_bundle` folder the moment I get that error.

**Comment by abernix** [comment]
@sebakerckhof Do you want to see if this is resolved for you in `1.4-beta.11`?

**Comment by sebakerckhof** [comment]
Yes, seems fixed.
